diff --git a/profiles/utexas/modules/contrib/ckeditor/includes/ckeditor.utils.js b/profiles/utexas/modules/contrib/ckeditor/includes/ckeditor.utils.js
index 90ee058..97aa276 100644
--- a/profiles/utexas/modules/contrib/ckeditor/includes/ckeditor.utils.js
+++ b/profiles/utexas/modules/contrib/ckeditor/includes/ckeditor.utils.js
@@ -161,8 +161,8 @@ if (typeof window.CKEDITOR_BASEPATH === 'undefined') {
     if (!CKEDITOR.env.isCompatible) {
       return;
     }
-
-    if (run_filter && ($("#" + textarea_id).val().length > 0) && typeof(ckeditor_obj.input_formats[ckeditor_obj.elements[textarea_id]]) != 'undefined' && ((ckeditor_obj.input_formats[ckeditor_obj.elements[textarea_id]]['ss'] == 1 && typeof(Drupal.settings.ckeditor.autostart) != 'undefined' && typeof(Drupal.settings.ckeditor.autostart[textarea_id]) != 'undefined') || ckeditor_obj.input_formats[ckeditor_obj.elements[textarea_id]]['ss'] == 2)) {
+    // Patch to prevent caching when no XSS response.
+    if (false && run_filter && ($("#" + textarea_id).val().length > 0) && typeof(ckeditor_obj.input_formats[ckeditor_obj.elements[textarea_id]]) != 'undefined' && ((ckeditor_obj.input_formats[ckeditor_obj.elements[textarea_id]]['ss'] == 1 && typeof(Drupal.settings.ckeditor.autostart) != 'undefined' && typeof(Drupal.settings.ckeditor.autostart[textarea_id]) != 'undefined') || ckeditor_obj.input_formats[ckeditor_obj.elements[textarea_id]]['ss'] == 2)) {
       $.ajax({
         type: 'POST',
         url: Drupal.settings.ckeditor.xss_url,
